tests/test_plotting.py

```diff
@@ -835,14 +835,18 @@ def debug_orders(pl, point):
         assert 2 in selected
 
 
+@requires("pyvista")
 def test_plotter_input(mapdl, make_block):
-    pl = Plotter(off_screen=True)
+    from ansys.mapdl.core.plotting.plotting import MapdlPlotter
+
+    pl = MapdlPlotter(off_screen=True)
+    pl_pv = pl._backend.pv_interface.scene
```

Comment:
I strongly believe we should have something like ``MapdlPlotter.backend_plotter``.